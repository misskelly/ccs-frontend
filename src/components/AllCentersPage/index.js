import React from 'react'

export const AllCentersPage = () => {
  return (
    <div>
      
    </div>
  )
}

export default AllCentersPage;
