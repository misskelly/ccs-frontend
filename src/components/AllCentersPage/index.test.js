
import React from 'react';
import { shallow } from 'enzyme';
import AllCentersPage from '.';


describe('AllCentersPage', () => {
  let wrapper;

  it('should match the snapshot', () => {
    wrapper = shallow(<AllCentersPage />);

    expect(wrapper).toMatchSnapshot();
  });
});