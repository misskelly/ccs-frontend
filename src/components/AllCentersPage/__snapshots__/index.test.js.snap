// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllCentersPage should match the snapshot 1`] = `ShallowWrapper {}`;
